apps/frontend-manage/src/components/questions/QuestionEditModal.tsx

```diff
@@ -1096,13 +1127,16 @@ function QuestionEditModal({
                         <div className="mr-2 font-bold">
                           {t('manage.questionForms.maximumLength')}:
                         </div>
-                        <FastField
+                        <FormikNumberField
                           name="options.restrictions.maxLength"
-                          type="number"
-                          className="mr-2 bg-opacity-50 border rounded w-44 border-uzh-grey-100 h-9 focus:border-primary-40"
+                          className={{
+                            field:
+                              'mr-2 bg-opacity-50 border rounded w-44  h-9 focus:border-primary-40',
+                          }}
                           placeholder={t('manage.questionForms.answerLength')}
-                          min={0}
```

Comment:
why remove min? would make sense to be able to specify only positive integers here

Reply:
because the current `FormikNumberField` component does not accept a minimum prop... This is still an open TODO, but only makes sense to be implemented after the validation issue on first render has been resolved before.